vizro-ai/src/vizro_ai/plot/components/visual_code.py

```diff
@@ -72,19 +97,20 @@ def _post_process(self, response: Dict, df_code: str, *args, **kwargs) -> str:
         return self._clean_visual_code(code_snippet)
 
     @_log_time
-    def run(self, chain_input: str, df_code: str, chart_types: str) -> str:
+    def run(self, chain_input: str, df_code: str, chart_type: str, df: pd.DataFrame = None) -> str:
         """Run chain to get visual code.
 
         Args:
             chain_input: User input or intermediate question if needed.
             df_code: Code snippet of dataframe.
-            chart_types: Chart types.
+            chart_type: Chart types.
```

Comment:
May you search for `"chart types"` in the codebase. It seems like we should change it to `"chart type"` in several more places (mostly in comments and docstrings)